docs/usage/configuration-options.md

```diff
@@ -705,6 +705,12 @@ If you enabled `automerge` in the Renovate config, you can speed up the automerg
 Caution (fixed in GitLab >= 12.7): when this option is enabled it is possible due to a bug in GitLab that MRs with failing pipelines might still get merged.
 This is caused by a race condition in GitLab's Merge Request API - [read the corresponding issue](https://gitlab.com/gitlab-org/gitlab/issues/26293) for details.
 
+## gitLabIgnoreApprovals
+
+Ignore the default project level approval(s), so that Renovate bot can automerge its merge requests, without needing approval(s).
```

Comment:
I think this will require `maintainer` rights on the target repo? If yes, we should note that here.

I wonder if a normal developer can simply overide approval rules. 🤔 

We should at least note that this only works for `Premium`  and higher GitLab tiers

Reply:
By default, this is allowed, but there is a setting to prevent overriding approval rules. It's probably worth mentioning that you shouldn't turn this on if you want to use this flag. https://docs.gitlab.com/ee/user/project/merge_requests/approvals/settings.html#prevent-overrides-of-default-approvals

I also don't believe this is a premium feature. The documentation is a little unclear. The Approval Settings page says All Tiers and Approval Rules says Premium. I can confirm that on Starter, you're allowed to set a single approval rule for merge requests and Merge Request creators can edit/remove the approval rule.